.github/workflows/release.yml

```diff
@@ -0,0 +1,102 @@
+        run: |
+          export PATH=/usr/lib/llvm-8/bin:$PATH
+          export CC=clang
+          export CXX=clang++
+          export ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk-bundle
+
+          # Build and create the artifacts
+          export current_release_tag=$(git describe --tags `git rev-list --tags --max-count=1`)
+          ./bazelw build \
+            --config=release-android \
+            --fat_apk_cpu=x86 \
+            --define=pom_version=${current_release_tag:1} \
+            //:android_deploy
+      - name: 'Configure gpg signing'
+        env:
+          GPG_KEY: ${{ secrets.gpg_key }}
+          GPG_KEY_NAME: ${{ secrets.gpg_key_name }}
+          GPG_PASSPHRASE: ${{ secrets.gpg_passphrase }}
+        run: |
+          # https://github.com/keybase/keybase-issues/issues/2798
+          export GPG_TTY=$(tty)
+          # Import gpg keys and warm the passphrase to avoid the gpg
+          # passphrase prompt when initating a deploy
+          # `--pinentry-mode=loopback` could be needed to ensure we
+          # suppress the gpg prompt
+          echo $GPG_KEY | base64 --decode > signing-key
+          gpg --passphrase $GPG_PASSPHRASE --batch --import signing-key
+          shred signing-key
+
+          gpg --pinentry-mode=loopback --passphrase $GPG_PASSPHRASE -ab dist/envoy.aar
+          gpg --pinentry-mode=loopback --passphrase $GPG_PASSPHRASE -ab dist/envoy-pom.xml
+          gpg --pinentry-mode=loopback --passphrase $GPG_PASSPHRASE -ab dist/envoy-javadoc.jar
+          gpg --pinentry-mode=loopback --passphrase $GPG_PASSPHRASE -ab dist/envoy-sources.jar
+      - name: 'Configure git'
+        run: |
+          # Switch global user to be envoy-bot for deployment
+          git config --global user.email "envoy-bot@users.noreply.github.com"
+          git config --global user.name "envoy-bot"
+      - name: 'Create sonatype staging profile id'
+        env:
+          SONATYPE_USER: ${{ secrets.sonatype_user }}
+          SONATYPE_PASSWORD: ${{ secrets.sonatype_password }}
+          ENVOY_PROXY_PROFILE_ID: ${{ secrets.envoy_proxy_profile_id }}
+        run: |
+          # Open release candidate with a specified profile id using `current_release_tag`
+          repository_id=$(mvn -s bazel/settings.xml org.sonatype.plugins:nexus-staging-maven-plugin:1.6.8:rc-open -DrepositoryId="sonatype-staging" -DstagingProfileId=$ENVOY_PROXY_PROFILE_ID -f dist/envoy-pom.xml | grep "\[INFO\] Opened" | cut -d " " -f3)
+      - name: 'Deploy with gpg signing'
+        env:
+          GPG_KEY: ${{ secrets.gpg_key }}
+          GPG_KEY_NAME: ${{ secrets.gpg_key_name }}
+          GPG_PASSPHRASE: ${{ secrets.gpg_passphrase }}
+          ENVOY_PROXY_PROFILE_ID: ${{ secrets.envoy_proxy_profile_id }}
+        run: |
+          # Sign and deploy to sonatype with the specified profile id
+          mvn -s bazel/settings.xml \
+            deploy:deploy-file \
+            -f dist/envoy-pom.xml \
+            -Dfile=envoy.aar \
```

Comment:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:3.0.0-M1:deploy-file (default-cli) on project envoy: /home/runner/work/envoy-mobile/envoy-mobile/dist/dist/envoy.aar not found. -> [Help 1]